BackendBench/constants.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD 3-Clause license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+Constants for BackendBench, including core operator lists.
+"""
+
+# The 77 core TorchBench operators
+# This list is derived from analysis of which operators appear most frequently
+# in TorchBench workloads and are considered high-priority for optimization
+TORCHBENCH_CORE_OPS = [
```

Comment:
@Laurawly @msaroufim similarly is it useful to say what a core op is in the test set?